### PR TITLE
feat: disable autocomplete on other form fields

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR
don't allow password manager to fill password field other then fields on unlock screen add types to text fields

### Link this PR to an issue [optional]

Fixes  #2480 new pr for #2497  

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
manually checking auto complete on firefox

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: src/app/screens/LNURLPay/index.tsx [hunk 1]
```diff
@@ -497,6 +498,7 @@ function LNURLPay() {
                           <div className="mt-4">
                             <TextField
                               id="name"
+                              type="text"
```
Comment (bumi) [inline src/app/screens/LNURLPay/index.tsx:501]: I think type text is default

File: src/app/screens/connectors/ConnectLnd/index.tsx [hunk 2]
```diff
@@ -175,6 +176,7 @@ export default function ConnectLnd() {
         <div>
           <TextField
             id="macaroon"
+            type="text"
```
Comment (bumi) [inline src/app/screens/connectors/ConnectLnd/index.tsx:179]: let's make this also a password field

File: src/app/screens/connectors/ConnectMyNode/index.tsx [hunk 3]
```diff
@@ -133,6 +133,7 @@ export default function ConnectMyNode() {
       <div className="mt-6">
         <TextField
           id="lndconnect"
+          type="text"
```
Comment (bumi) [inline src/app/screens/connectors/ConnectMyNode/index.tsx:136]: let's make this also a password field

File: src/app/screens/Send/index.tsx [hunk 4]
```diff
@@ -126,6 +126,7 @@ function Send() {
           <div className="pt-4">
             <TextField
               id="invoice"
+              type="text"
```
Comment (reneaaron) [inline src/app/screens/Send/index.tsx:129]: Why did you add the type everywhere? It's "text" by default?

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 5]
```diff
@@ -164,6 +165,7 @@ export default function ConnectRaspiBlitz() {
         <div>
           <TextField
             id="macaroon"
+            type="text"
```
Comment (bumi) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:168]: let's make this also a password field

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/app/screens/connectors/ConnectStart9/index.tsx [hunk 6]
```diff
@@ -136,6 +136,7 @@ export default function ConnectStart9() {
       <div className="mt-6">
         <TextField
           id="lndconnect"
+          type="text"
```
Comment (bumi) [inline src/app/screens/connectors/ConnectStart9/index.tsx:139]: let's make this also a password field. 

lndconnect holds secrets and thus we can handle it as a password field I guess.

File: src/app/screens/connectors/ConnectUmbrel/index.tsx [hunk 7]
```diff
@@ -133,6 +133,7 @@ export default function ConnectUmbrel() {
       <div className="mt-6">
         <TextField
           id="lndconnect"
+          type="text"
```
Comment (bumi) [inline src/app/screens/connectors/ConnectUmbrel/index.tsx:136]: let's make this also a password field

**Review by bumi — CHANGES_REQUESTED** [review]

File: src/app/screens/Send/index.tsx (hunk 4 above)
Comment (pavanjoshi914) [reply src/app/screens/Send/index.tsx:129]: we have type added as text at few of the text field and few of them doesn't contain them. so shall we remove type field at all the fields containing type="text"

it's better to be consistent!

Comment (pavanjoshi914) [reply src/app/screens/Send/index.tsx:129]: having type explicitly in the code would be better. isn't it?

File: src/app/screens/LNURLPay/index.tsx (hunk 1 above)
Comment (pavanjoshi914) [reply src/app/screens/LNURLPay/index.tsx:501]: we have type="text" at few of the text field and few of them doesn't contain them.  it's better to be consistent!

shall we remove it from everywhere or add it everywhere? isn't this better and readable to have type explicitly defined
